ops/docker-compose.yml

```
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  app:
+    image: pentacent/keila
+    ports:
+      - "4000:4000"
+    command:
+      - "start"
+    depends_on:
+      - db
+    build:
+      context: .
+    environment:
+      SECRET_KEY_BASE: "${YOUR_SECRET_KEY}"
+      DB_URL: "ecto://postgres:postgres@db/postgres"
```

```suggestion
      DB_URL: "postgres://postgres:postgres@postgres/postgres"
```
<- This is the best URL ever!

---

Naming is important so people know _exactly_ what you're talking about, I guess. It doesn't get any more postgres-y than this one, yeah 😅